src/app/ui/color_bar.cpp

```diff
@@ -824,6 +824,16 @@ void ColorBar::onTilesButtonClick()
                                             TilemapMode::Pixels));
 }
 
+void ColorBar::enableTilemapModeButton(bool state)
+{
+  m_tilesButton.setEnabled(state);
+}
+
```

Comment:
Oh, I see. I prefer to lock Tilemap Mode, otherwise an accidental `Spacebar + Tab` can mess up the tileset without notice it on the canvas. Do you agree?
For the moment, I modified `MovingPixelsState::onBeforeCommandExecution` to not drop the pixels if the command is `CommandId::ToggleTilesMode()` + `ToggleTilesModeCommand` is locked while the Tilemap Mode toggle button is disabled.